modules/rosetta/src/test/scala/org/tessellation/rosetta/domain/construction/ConstructionServiceSuite.scala

```diff
@@ -297,4 +301,18 @@ object ConstructionServiceSuite extends MutableIOSuite with Checkers with Transa
 
     testParseTransactions(isSignedTransaction = false, testCaseCallbackHandler = testCase)
   }
+
+  test("encode EncodedRosettaTransaction") { res =>
+    implicit val (sp, k) = res
+
+    generateTestTransactions(wantSignedTransaction = false).map { a =>
+      val (_, _, hashedTransaction) = a.head
+
+      val encoded = hashedTransaction.asJson
+      val decoded = encoded.as[Hashed[Transaction]]
+
+      println(decoded)
+      expect.eql(true, true)
```

Comment:
It will always succeed

Reply:
This test was to test the decoder and encoded. I can still not find the implicits for encoding and decoding `Hashed[Transaction]]` here:

https://github.com/cngo-github/tessellation/blob/20230307-base64/modules/rosetta/src/test/scala/org/tessellation/rosetta/domain/construction/ConstructionServiceSuite.scala#L311

Why?